docs/players/ruffle.md

```diff
@@ -2,4 +2,7 @@
 
 Ruffle is a flash emulator written in the Rust programming language. Ruffle can be used in the browser through the Ruffle [demo](https://ruffle.rs/demo/) by selecting an SWF file from your local files, or it can also be added to your personal website.
 
-All of this information and more can be accessed on the official [Ruffle](https://ruffle.rs/) website.
\ No newline at end of file
+All of this information and more can be accessed on the official [Ruffle](https://ruffle.rs/) website.
+
+!!! warning "Warning:"
```

Comment:
No need to have that `"Warning:" title.

Reply:
do i remove it or replce it with a note or smth